packages/chain-adapters/src/utxo/utxoSelect/index.test.ts

```diff
@@ -158,4 +178,43 @@ describe('utxoSelect', () => {
     }
     expect(result).toEqual(expectedResult)
   })
+  it('with passed `from` param in input - should return fee and no inputs/outputs in case there is no UTXO for the passed from address', () => {
```

Comment:
we probably need one or two additional test cases to ensure that we can send the correct `opReturnData` (i.e. the deposit/withdraw memo) in the outputs too.

Reply:
disregard this, `opReturnData` is handled by hdwallet signing the tx